tests/integrationv2/tox.ini

```diff
@@ -11,4 +11,4 @@ deps =
     pytest==5.3.5
     pytest-xdist
 commands =
-    pytest --cache-clear -rpfsq {env:TOX_TEST_NAME:""}
+    pytest -n8 --cache-clear -rpfsq {env:TOX_TEST_NAME:""}
```

Comment:
Curious what instance size this is intended for?
after some more runtime numbers, it would be good to make this value discoverable.

Reply:
xdist talks about tying this to the number of cpus, and I think optimizes for cpu bound cases. All our tests are I/O bound though, so I don't think there will be strong correlation of performance to cpu count. It actually looks like the bottleneck is in dealing with an increasing number of workers, e.g when I tried 24 workers the framework slowed down quite a bit.

It would be great to have a system that optimized for IO bound tests.